docs/spelling_wordlist.txt

```diff
@@ -21,6 +21,7 @@ cryptographic
 cryptographically
 Debian
 decrypt
+decrypts
```

Comment:
I'd like to register my (continued) displeasure that enchant can't figure out plurals.